src/main/java/Sirius.java

```diff
@@ -0,0 +1,148 @@
+    private static int taskCounter = 0;
+    private static boolean isExit = true;
+    private static final ArrayList<Task> list = new ArrayList<>();
+
+    // some methods
+    public static void sayHello(){
+        System.out.println("""
+                -----------------------
+                Hello! I'm Sirius!
+                What can I do for you?
+                -----------------------
+                """);
+    }
+    public static void sayGoodbye(){
+        System.out.println("""
+                -----------------------
+                Bye! Hope to see you soon.
+                -----------------------
+                """);
+        isExit = false;
+    }
+    public static void markTask(String[] commandPieces, ArrayList<Task> list, boolean isMarked) {
+        System.out.println(SAPERATOR);
+        int taskNumber = Integer.parseInt(commandPieces[1]);
+        if (taskNumber > taskCounter) {
+            System.out.println("Oh, please enter an valid task number!");
+        }
+        else{
+            if (isMarked){
+                list.get(taskNumber - 1).setMarked(true);
+                System.out.println("Nice! I've marked this task as done:");
+            }
+            else{
+                list.get(taskNumber - 1).setMarked(false);
+                System.out.println("OK, I've marked this task as not done yet:");
+            }
+            System.out.println(list.get(taskNumber - 1).toString());
+        }
+        System.out.println(SAPERATOR);
+    }
+    public static void addTask(String[] commandPieces, ArrayList<Task> list){
+        System.out.println(SAPERATOR);
+        String commandPrefix = commandPieces[0];
+        String taskName = commandPieces[1];
+
+        switch (commandPrefix){
+            case "deadline":
+                list.add(new Deadline(taskName, false, commandPieces[2]));
+                break;
+            case "event":
+                list.add(new Event(taskName, false, commandPieces[2], commandPieces[3]));
+                break;
+            default:
+                list.add(new Todo(taskName, false));
+        }
+        System.out.println("Got it. I've added this task:");
+        System.out.println(list.get(taskCounter).toString());
+        taskCounter++;
+        System.out.println("Now you have " + taskCounter + " tasks in the list.");
+        System.out.println(SAPERATOR);
+    }
+    public static void listTasks(ArrayList<Task> list){
+        System.out.println(SAPERATOR);
+        System.out.println("Here are the tasks in your list:");
+        for (int i = 0; i < taskCounter; i++) {
+            System.out.print(i + 1 + ". ");
+            System.out.println(list.get(i).toString());
+        }
+        System.out.println(SAPERATOR);
+    }
+    public static String[] splitCommand(String userInput){
+        String[] slashCommand = userInput.split(SLASH);
+        String commandContent = slashCommand[0];
+        String commandPrefix = commandContent.split(SPACE)[0];
```

Comment:
I think it will be good to comment on what these do because down the code it gets confusing. maybe comment (clarify) here so readers who are unfamiliar with your code can understand what these variables meant quickly.